packages/cli/src/Init.ts

```diff
@@ -111,6 +121,7 @@ export class Init implements Command {
              -h, --help   Display this help message
   --datasource-provider   Define the datasource provider to use: postgresql, mysql, sqlite, sqlserver, mongodb or cockroachdb
                   --url   Define a custom datasource url
+           --with-model   Add example model to created schema file
```

Comment:
Turns out, this is of course a flag - and not an option:
```suggestion

    ${bold('Flags')}

           --with-model   Add example model to created schema file
```

(Also, you could add an empty line between line 118 and 119 - that seems to be missing for a long time!)

Reply:
Done! Thanks a lot for your help!